addons/sys_rack/stringtable.xml

```diff
@@ -13,6 +13,9 @@
             <English>Radio used externally. Only owner may configure.</English>
             <Japanese>無線機は外部で使われています。オーナーのみが設定可能です。</Japanese>
         </Key>
+        <Key ID="STR_ACRE_sys_radio_alreadyOpenRadio">
+            <English>Radio interface is opened by another player.</English>
```

Comment:
Too immersion breaking! 😆 
Something like "Radio in hand of an external user." might be better.